projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -211,10 +217,11 @@ const ProtectAdminPage = () => {
 												marginLeft: 0,
 											} }
 										/>
-										<span>{ __( 'Scanning your site…', 'jetpack-protect' ) }</span>
+										<span>{ heading }</span>
 									</Col>
 									<Col>
 										<H3>{ __( 'Your results will be ready soon', 'jetpack-protect' ) }</H3>
+										{ currentProgress >= 0 && <ProgressBar value={ currentProgress } /> }
```

Comment:
```suggestion
										{ { currentProgress !== null && currentProgress >= 0 && <ProgressBar value={ currentProgress } /> }
```

We default an unavailable scan progress to `null`, which is (tragically) considered larger than or equal to zero 😅

```javascript
null >= 0 === true // JS weirdness
```